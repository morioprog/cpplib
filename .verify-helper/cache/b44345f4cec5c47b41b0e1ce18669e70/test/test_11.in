5
-2 1
9 7
-2 7
8 1
5 1
