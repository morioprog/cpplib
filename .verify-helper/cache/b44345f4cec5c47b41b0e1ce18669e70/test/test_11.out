8 1
5 1
9 7
-2 7
-2 1
